integration-tests/src/test/java/org/zowe/apiml/startup/impl/ApiMediationLayerStartupChecker.java

```diff
@@ -95,7 +95,7 @@ private boolean areAllServicesUp() {
             }
             String allComponents = context.read("$.components.discoveryComposite.components.discoveryClient.details.services").toString();
             boolean isTestApplicationUp = allComponents.contains("discoverableclient");
-            boolean isCloudGatewayUp = allComponents.contains("cloudgateway");
+            boolean isCloudGatewayUp = allComponents.contains("cloud-gateway");
```

Comment:
Probably it is good also update the value at `org.zone.apiml.util.service.FullApiMediationLayer#prepareCloudGateway`

Reply:
done